docs/docs/50-running/55-compute-resources.md

```diff
@@ -36,10 +36,12 @@ This sets all workloads in the `foo` acorn to have `256Mi` of memory except for
 To set a compute class at run time, you can utilize the `--compute-class` flag.
 
 :::note
-Check out the [compute class reference documentation](100-reference/06-compute-resources.md#compute-classes) for more information about how workload classes work.
+Check out the [compute class reference documentation](100-reference/06-compute-resources.md#compute-classes) for more information about how compute classes work.
 :::
 
-When setting this value, you have two options - globally or per workload and you can do a combination of both. When setting the memory globally for that Acorn, you just define the compute class you would like to set.
+### --compute-class
+
+When setting this compute classes, you have two options - globally or per workload and you can do a combination of both. When setting the memory globally for that Acorn, you just define the compute class you would like to set.
```

Comment:
`When setting the compute classes, you have two options - globally or per workload, and you can do a combination of both. When setting the memory globally for that Acorn, you just define the compute class you would like to set. Any classes defined for a specific container or job will overwrite the global value.`